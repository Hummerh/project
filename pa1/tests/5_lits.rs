b'H' b'\'' b'\\';
b"Hello" b" \"\\\" " b"123 abc";
123 123_456;
true false;
