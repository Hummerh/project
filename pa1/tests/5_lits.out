LITCHAR(H)
LITCHAR(')
LITCHAR(\)
;
LITSTR(Hello)
LITSTR( "\" )
LITSTR(123 abc)
;
LITDEC(123)
LITDEC(123456)
;
LITBOOL(true)
LITBOOL(false)
;
